api/proto/info/info.proto

```diff
@@ -539,8 +539,9 @@ message ZInfoVolume {
   VolumeResources resources = 4;
   ZSwState state = 5;
   uint32 progressPercentage = 6; // 0-100% Use depends on state i.e., download vs. verification vs. volume creation
+  int64 generation_count = 7; // version of volume
 
-  ErrorInfo volumeErr = 7;
+  ErrorInfo volumeErr = 8;
```

Comment:
Is this already used in the controller? If so, it is better to not change the tag but to make the generation_count be 8

Reply:
Resolved